common-custom-user-data-maven-extension/pom.xml

```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>com.gradle</groupId>
             <artifactId>gradle-enterprise-maven-extension</artifactId>
-            <version>1.10.4</version>
+            <version>1.11-branch-francois_maven_test-prerelease</version>
```

Comment:
Since this is used as a template for customers, we should avoid having unreleased versions merged into `master`. I guess we can keep this PR in a branch until `1.11` is released.

Reply:
yes, that was the plan. I'll change the PR to contain 1.11 when released and we will merge it after. Keeping it like that so that you can play around with the custom 1.11 for now